src/js/components/RadioButton/stories/Theme.js

```diff
@@ -22,6 +22,9 @@ const theme = deepMerge(grommet, {
     icon: {
       size: '10px',
     },
+    font: {
+      weight: 500,
+    },
```

Comment:
Thank you for adding that!

Reply:
No problem, the codebase is very clean and easy to follow 🙂